geode-core/src/integrationTest/java/org/apache/geode/cache30/ShorteningExpirationTimeRegressionTest.java

```diff
@@ -53,67 +48,52 @@
   private static final int LONG_WAIT_MS = 2 * 60 * 1000;
   private static final int SHORT_WAIT_MS = 1;
 
-  private static final String KEY = "key";
-
-  private String uniqueName;
-  private String regionName;
+  @ClassRule
+  public static ServerStarterRule server = new ServerStarterRule().withAutoStart();
```

Comment:
I like the old way this test setup and teared down the cache for each test.
Any reason for changing that?
I think your actual fix for this test was deleting line 100 which asserted that the value was "quickExpire" which might not be true because the expire could have already happened.

I like the tests using the actual product APIs. The rules obscure what is happening. In this case I would guess that ServerStarterRule starts up a cache server which is overkill for this test. All this test needs is a cache.

Why switch to "@ClassRule" and "@BeforeClass"? Was it just to save some time? I guess that is okay.